app/assets/stylesheets/application/_layout.scss

```diff
@@ -104,3 +107,58 @@ pre {
   font-size: 14px;
   line-height: 1.4;
 }
+
+.mumuki {
+  text-align: center;
+  h1 {
+    font-size: 45px;
+    margin: 0;
+
+    .ahahamojimoji {
+      font-size: 180px;
+    }
+  }
+
+  h2 {
+    margin: 15px 0px;
+    font-size: 35px;
+  }
+
+  .logo {
+    margin-top: -95px;
+  }
+}
+
+
+.user-assignments {
+  margin-top: 20px;
+}
+
+.progress-list-flex {
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 20px;
+  a {
+    border-width: 1px;
+    border-color: white;
+    border-style: solid;
+    width: 100px;
```

Comment:
We should move the progress listing css to another file. We should also do that for the landing page styles